### DISCUSSION TITLE
Allow to hide specific achievements?

### DISCUSSION THREAD

**Discussion by Jackenmen** [community/community#18168] [Other Feature Feedback, Questions, & Ideas] [answered]
Now that the achievements have been introduced, it's no longer possible to granularly disable the individual badges which used to be possible with Mars 2020 Contributor and Arctic Code Vault Contributor badges.

**Comment by antonio-hickey** [comment]
Yea it would be nice to be able to select specific achievements to hide/share this could get congested, and down play cool achievements like the mars flight.

**Comment by axrobert** [comment]
It would be nice if this were possible, just put achievements you want.

**Comment by Jackenmen** [comment, marked as answer]
This has been addressed: https://github.blog/changelog/2022-10-07-visibility-settings-for-individual-achievements/

**Reply by antonio-hickey** [reply]
:+1:
